plugins/scanners/dos/README.md

```diff
@@ -0,0 +1,27 @@
+# Double Open Server (DOS) Scanner Wrapper
+
+The DOS scanner wrapper is a client for [DOS] to initiate scans on the server-side, performed by [ScanCode], and to reuse scan results on a per-file basis.
+This effective implements "delta-scanning" of only the changed files between package versions.
```

Comment:
"effectively"
As the scanner uses its own storage, should this mention that no other storage writer should be configured?

Reply:
The implementation hard-codes `readFromStorage = false, writeToStorage = false`, so I've added a sentence about that.